deployment/media-insights-stack.yaml

```diff
@@ -724,7 +825,7 @@ Resources:
     DeletionPolicy: Retain
     Properties:
       CompatibleRuntimes:
-        - python3.7
+        - python3.6
```

Comment:
Is this intentional? I see 3.7 listed below for the layer

Reply:
I wasn't sure where this change came from, but I left it because this is the python3.6 lambda layer.   I guess it could be used with a Python3.7 lambda, though.  We could set it to both, but I think setting it to 3.7 only will make it unusable in a Python 3.6 lambda.

Reply:
I misread this section. Looks good now that I have reviewed it again.